onnxruntime/core/providers/rocm/tunable/gemm_common.h

```diff
@@ -55,6 +55,7 @@ struct GemmParams : tunable::OpParams {
   T beta;
   T* c;
   int64_t ldc;
+  int64_t batch = 1;
```

Comment:
Why is batch is need in `GemmParams`, instead of `BatchedGemmParams` below?

Reply:
Just that `params->batch` is wanted for batched case while `GemmParams` doesn't have it.

Reply:
Reverted.